### DISCUSSION TITLE
PrismaClient is unable to run

### DISCUSSION THREAD

**Discussion by asmegithub** [prisma/prisma#22143] [Q&A]
### Question

when I tried to fetch the data from database using prisma orm it says Error: PrismaClient is unable to run in this browser environment, or has been bundled for the browser (running in `unknown`).  please help me guys.

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by nurul3101** [comment]
Hello @asmegithub 👋 

The error message you're encountering typically indicates that PrismaClient is being instantiated or used in a context that is not supported, such as client-side code in a web browser.

Are you using Next.js? If yes, are you using `PrismaClient` in API routes or server-side functions like `getServerSideProps` or `getStaticProps` and not in the components or pages that are rendered on the client-side? 

Can you please also take a look at this https://github.com/prisma/prisma/issues/6219. It has some solutions to this same problem. Also, you might want to look at this [documentation](https://www.prisma.io/docs/guides/other/troubleshooting-orm/help-articles/nextjs-prisma-client-dev-practices) for instantiating Prisma Client with Next.js

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
